internal/testsuite/eventwatcher/eventwatcher.go

```diff
@@ -210,11 +209,10 @@ func GetFromIngresses(parent context.Context, C chan *api.EventMessage) error {
 }
 
 func getFromIngress(ctx context.Context, host string) error {
-	tokens := strings.Split(host, ".")
-	if len(tokens) == 0 {
-		return errors.Errorf("malformed hostname: %s", host)
+	ingressUrl := os.Getenv("ARMADA_EXECUTOR_INGRESS_URL")
```

Comment:
It seems like you are missing the case of a bad hostname from host parameter.  

Is that intentional?

Reply:
What do you mean?

Reply:
I think that's fine.

Reply:
Kk.  I didn't see that this was in test suite.  Yea I think that's fine.